Formula/acmetool.rb

```diff
@@ -1,11 +1,10 @@
-require "language/go"
-
 class Acmetool < Formula
   desc "Automatic certificate acquisition tool for ACME (Let's Encrypt)"
   homepage "https://github.com/hlandau/acme"
   url "https://github.com/hlandau/acme.git",
       :tag      => "v0.0.67",
       :revision => "221ea15246f0bbcf254b350bee272d43a1820285"
+  revision 1
 
   bottle do
```

Comment:
Another quick question: when I bump revision, should I also remove `rebuild` or leave that for the bot?

Reply:
You can leave the entire `bottle do` block untouched. The CI will handle that during the merge process.